test/pseudo-tty/test-prompt.in

```diff
@@ -0,0 +1,2 @@
+sushi
```

Comment:
Can you test with non-ASCII characters please?

Reply:
I have tried one time, but it seems throwing confusing error. I'd try again in my next PR.

Reply:
My guess is that is overflows the `Uint8Array`, as non-ASCII char typically takes more than one byte.

Reply:
It seems that implementation works in REPL well, but fails in test. My guess is that `pseudo-tty` test may not support non-ASCII character as input? I will fill an issue, if I can confirm that with current API.

REPL
```
Welcome to Node.js v17.0.0-pre.
Type ".help" for more information.
> prompt("hello")
hello
寿司
'寿司'
> 
```

Test
```
[00:00|%   0|+   0|-   0]: release test-prompt match failed
line=0
expect=^\�\�\�\�\�\�$
actual=寿司
=== release test-prompt ===                   
Path: pseudo-tty/test-prompt
寿司
what your favorite food?
Command: /System/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python /Users/qingyudeng/projects/node/test/pseudo-tty/pty_helper.py out/Release/node /Users/qingyudeng/projects/node/test/pseudo-tty/test-prompt.js
[00:00|% 100|+   0|-   1]: Done         
```